rosbag2_storage/include/rosbag2_storage/storage_interfaces/base_info_interface.hpp

```diff
@@ -39,6 +39,18 @@ class ROSBAG2_STORAGE_PUBLIC BaseInfoInterface
    * \returns the relative path.
    */
   virtual std::string get_relative_file_path() const = 0;
+
+  /**
+   * Returns the size of the bagfile.
+   * \returns the size of the bagfile in bytes.
+   */
+  virtual uint64_t get_bagfile_size() const = 0;
```

Comment:
doesn't #197 state to have bagfile size part of the `IO` interface?

Reply:
I think I accidentally re-worded the issue and flipped the function migration.
I'll re-review this work in a few hours

Reply:
Update:
The correct wordage should be: "`get_bagfile_size` and `get_storage_identifier` should be moved to BaseInfoInterface".

Reply:
Update:
Corrected wordage in issue